paddle/fluid/pybind/inference_api.cc

```diff
@@ -249,13 +252,37 @@ void BindAnalysisConfig(py::module *m) {
       .def("cpu_math_library_num_threads",
            &AnalysisConfig::cpu_math_library_num_threads)
       .def("to_native_config", &AnalysisConfig::ToNativeConfig)
+      .def("enable_quantizer", &AnalysisConfig::EnableMkldnnQuantizer)
+      .def("quantizer_enabled", &AnalysisConfig::mkldnn_quantizer_enabled)
+      .def("quantizer_config", &AnalysisConfig::mkldnn_quantizer_config,
```

Comment:
Please, wrap this `def` with `#ifdef PADDLE_WITH_MKLDNN`

Reply:
Done